src/Hazelcast.Net/CP/CPSubsystem.cs

```diff
@@ -93,15 +93,19 @@ public static (string groupName, string objectName) ParseName(string name)
             }
             else
             {
-                groupName = name.Substring(pos + 1).Trim();
+                groupName = name[(pos + 1)..].Trim();
                 if (groupName.Equals(DefaultGroupName, StringComparison.OrdinalIgnoreCase))
                     groupName = DefaultGroupName;
             }
 
             if (groupName.Length == 0)
                 throw new ArgumentException("CP group name cannot be an empty string.", nameof(name));
 
+#if NETSTANDARD2_1
+            if (groupName.Contains("@", StringComparison.OrdinalIgnoreCase))
```

Comment:
because 2.1 made it available to specify the comparison, and then the recommendation is to specify it

Reply:
What if we define `ContainsNoCase` extension method, for 2.1 forward it to framework implementation, otherwise use something [like this](https://stackoverflow.com/a/63371269)?

I think extension methods like `ContainsNoCase`, `IndexOfInvariant`, etc. are simpler than adding `StringComparison` parameter every time. In our case it will also help as of adding this `#if` blocks everywhere.

Reply:
check what I've done, it was done for other methods: there is a `StringNetStandardExtensions` class that defines the missing methods for NetStandard 2.0 - so you can code just as if you had 2.1 - so your code is future proof. what do you think?

Reply:
So you plan add `Contains` to `StringNetStandardExtensions` also?